lib/cc/analyzer/engine.rb

```diff
@@ -52,11 +51,6 @@ def run(stdout_io, container_listener)
         container.run(container_options).tap do |result|
           CLI.debug("#{qualified_name} engine stderr: #{result.stderr}")
         end
-      rescue Container::ImageRequired
```

Comment:
A different exception is thrown by the `Registry` itself, and it has the context there so there's no need to catch and re-raise.